ds3-autogen-net/src/main/resources/tmpls/net/request/common/optional_checksum.ftl

```diff
@@ -1,14 +1,14 @@
         private ChecksumType _checksum = ChecksumType.None;
-        private ChecksumType.Type _type;
+        private ChecksumType.Type _ctype;
```

Comment:
The problem is that there is a ChecksumType and a ChecksumType.Type, which causes infinate confusion. Also, _checksumType would correspond with ChecksumType, which is a class, so I can't use that.